stackslib/src/chainstate/stacks/boot/mod.rs

```diff
@@ -843,6 +873,7 @@ impl StacksChainState {
                 missed_reward_slots: missed_slots,
             },
             signers: signer_set,
+            pox_stx_threshold: Some(threshold),
```

Comment:
Maybe this should be `pox_ustx_threshold`?

Reply:
Done!